lib/rubocop/cop/rspec/focus.rb

```diff
@@ -30,8 +30,13 @@ class Focus < Base
                           Examples::PENDING).node_pattern_union
 
         def_node_matcher :metadata, <<-PATTERN
-          {(send #rspec? #focusable_selector? <$(sym :focus) ...>)
-           (send #rspec? #focusable_selector? ... (hash <$(pair (sym :focus) true) ...>))}
+          (send #rspec? #focusable_selector?
+            {
+              <$(sym :focus) ...>
```

Comment:
`< >` is necessary, some specs fail if it's removed.
The same for `pending` and `multiple_expectations`: I'm not sure if it's an inadequate test coverage, or for `... hash` it's not needed.

Reply:
I think this is an example where readability is not  being improved